skate-plugin/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ intellij {
   plugins.add("org.intellij.plugins.markdown")
   plugins.add("org.jetbrains.plugins.terminal")
   plugins.add("org.jetbrains.kotlin")
+  plugins.add("org.jetbrains.android")
```

Comment:
I'm surprised we don't need the google-made one. Are these the same?

Reply:
Which google-made plugin you're referring to? I'm adding this one because the SyncListener is part of if
https://github.com/JetBrains/android/tree/master